### DISCUSSION TITLE
I see some models transparent

### DISCUSSION THREAD

**Discussion by johnwwk** [google/model-viewer#3873] [Q&A]
### Description
Hi, as you see from the video, the model appears transparent.  This happens especially with the GLB models I download from Sketchfab, and happens very commonly.
Is there a setting I am missing with the component, is this a bug or the problem with the model itself? 
If it is a problem with the model, what do you recommend overcoming the issue?
Thank you
### Live Demo
https://ipfs.motif.foundation/ipfs/QmbcVMYC38sLX8h7f1TqmM2hp1vXShKpDFkKatGjHoKE6R
### Version
"@google/model-viewer": "1.10.0",
### Browser Affected 
- [x ] Chrome, latest
### OS
- [x ] MacOS 

### Code
   ```
 mediaStyle.position = "sticky";
    mediaStyle.background = "linear-gradient(0deg, #FFDEE9 0%, #B5FFFC 100%)";
    mediaStyle.posterColour = "linear-gradient(0deg, #FFDEE9 0%, #B5FFFC 100%)";
    mediaStyle.pointerEvents = "auto";
 
       <model-viewer
            bounds="tight"
            style={mediaStyle}
            id="previewModel"
            src={props.asset.metadata.artwork ? null : encodeURI(props.asset.contentURI)}
            interaction-prompt={"none"}
            poster={props.asset.metadata.artwork || null}
            loading={"lazy"}
            seamless-poster
            disable-zoom
            environment-image="neutral"
            shadow-intensity="1"
          ></model-viewer>
```

**Comment by elalish** [comment]
That definitely looks like an error in the GLB; can you try it in some other viewers? Also, if you open it in our [editor](https://modelviewer.dev/editor/), that will give some details on the model and any possible errors/warnings.

**Comment by jboe182** [comment]
If you use a image texture with less then 100% Opacity it will texture the GLB Model as translucent.

I did this for glass elements in AR.
